edisch/drupal-dashboard-module
# 7: Posting to posttestserver.com and printing response to the menu.

Steps to test

Go to http://drupal7.dev/examples/requirements which will issue a POST request to http://posttestserver.com
Expect a http response from the POST request formatted in JSON to be printed
There will be a url in the body of the response from the http://posttestserver.com which you can follow to see the details of the POST request issued.
